src/components/About/AbouCard.jsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Typography } from '@mui/material'
+import Button from '@mui/material/Button'
+import Card from '@mui/material/Card'
+import CardActions from '@mui/material/CardActions'
+import CardContent from '@mui/material/CardContent'
+import CardMedia from '@mui/material/CardMedia'
+
+const AboutCard = ({ item, mobileActive, seeMore }) => {
+  return (
+    <Box mb={mobileActive ? 4 : 0} mx={{ xs: 1, sm: 0 }}>
+      <Card
+        sx={{
+          borderRadius: '0px',
+        }}
+      >
+        <CardMedia
+          component="img"
+          alt={item.title}
+          height={'170'}
+          image={item.path}
+          sx={{
+            userSelect: 'none',
+          }}
+        />
+        <CardContent
+          sx={{
+            backgroundColor: '#080d26',
+            color: 'white',
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            height: '100px',
+            width: '100%',
+          }}
+        >
+          <Typography
+            gutterBottom
+            variant="h7"
+            component="div"
+            align="center"
+            mt={2}
+            sx={{
+              fontFamily: 'Montserrat',
```

Comment:
For the font family--the globals are described as such. Not sure if Montserrat is part of the build.

Global css (body):
>   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;